lib/oli_web/templates/layout/authoring.html.leex

```diff
@@ -11,20 +11,9 @@
     <link rel="icon" type="image/png" sizes="32x32" href="<%= favicons("favicon-32x32.png") %>">
     <link rel="manifest" href="/site.webmanifest">
 
-    <!-- This page supports both dark and light color schemes, defaults to light. -->
-    <meta name="color-scheme" content="light dark">
-
-    <!-- Janus Part Component (Slider & Carousel) external CSS File -->
-    <link rel="stylesheet" href="/css/janus_input_number_delivery.css"/>
-    <link rel="stylesheet" href="/css/janus_slider_delivery.css"/>
-    <link rel="stylesheet" href="/css/janus_image_carousel_delivery.css"/>
-
     <!-- Theme CSS (contains all bootstrap styles as well as custom styles for a specific theme) -->
-    <link id="authoring-theme-light" rel="stylesheet" href="/css/authoring_torus_light.css" media="screen and (prefers-color-scheme: light)"/>
-    <link id="authoring-theme-dark" rel="stylesheet" href="/css/authoring_torus_dark.css" media="screen and (prefers-color-scheme: dark)"/>
-
-    <link id="delivery-theme-light" rel="stylesheet" href="/css/delivery_torus_light.css" media="(prefers-color-scheme: light)"/>
-    <link id="delivery-theme-dark" rel="stylesheet" href="/css/delivery_torus_dark.css" media="(prefers-color-scheme: dark)"/>
+    <link id="authoring-theme" rel="stylesheet" href="/css/authoring_default.css" />
```

Comment:
there is a bit of hackery that no longer works because the adapative author doesn't have a dark mode yet
https://github.com/Simon-Initiative/oli-torus/blob/master/assets/src/apps/authoring/Authoring.tsx#L128

Reply:
Thanks Ben, I believe I have addressed this with https://github.com/Simon-Initiative/oli-torus/pull/2389/commits/48b0b1694081e1d7d730eec868d8bd9ef1fc9cf3 but please let me know if you still see an issue